artifacthub-pkg.yml

```diff
@@ -1,67 +1,50 @@
----
-version: 0.1.4
+# Kubewarden Artifacthub Package config
+#
+# Use this config to submit the policy to https://artifacthub.io.
+#
+# This config can be saved to its default location with:
+#   kwctl scaffold artifacthub > artifacthub-pkg.yml 
+version: 0.1.4-11-gd2e715c
```

Comment:
This should be fixed. This kind of version should be used only in development phase. If we merge this, the released version will be wrong.